WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadUtils.java

```diff
@@ -202,6 +203,14 @@ private static void showSnackbarSuccessAction(View view, int messageRes, int but
                 .show();
     }
 
+    private static void showSnackbarSuccessAction(View view, String messageRes, int buttonTitleRes,
```

Comment:
The param `messageRes` is misnamed since it's not a message resource. Also, the existing version of this routine which *does* have a message resource param should simply call this new routine.

Reply:
addressed in 8f0a327